### ISSUE TITLE
Non-existing restaurant stored in session throws Exception

### ISSUE THREAD

**Issue by alexsegura** [coopcycle/coopcycle-web#418]
```
Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Argument 1 passed to AppBundle\Sylius\Order\OrderFactory::createForRestaurant() must be an instance of AppBundle\Entity\Restaurant, null given, called in /var/www/demo/releases/124/src/AppBundle/Sylius/Cart/RestaurantCartContext.php on line 56 in /var/www/demo/releases/124/src/AppBundle/Sylius/Order/OrderFactory.php:34
```

**Comment by AmelieA** [comment]
That is a really pressing isue. We're going to have a lot of those on 1st october

**Comment by alexsegura** [comment]
Actually, this is a relatively edge-case. This happens in the demo, because we clear the database every day, so a returning visitor may have a non-existing restaurant in session, because the identifiers have changed during the night. 

But it definitely needs to be addressed.

**Comment by alexsegura** [comment]
Fixed in 6fe7437b0de4a6b0d1d451d51f19d1df6464dbf4
